### PR TITLE
d2: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/d2.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`d2 themes`
+
+- Use a different [t]heme for the output file (list available themes first to get the desired `theme_id`):
+
+`d2 --theme {{theme_id}} {{in.d2}} {{out.svg|out.png}}`
+
+- Make rendered diagrams look like hand [s]ketches:
+
+`d2 --sketch true {{in.d2}} {{out.svg|out.png}}`
```
Comment (kbdharun) [inline pages/common/d2.md:28]:
```suggestion
`d2 {{path/to/input_file.d2}} {{output_file.svg|output_file.png}}`

- [w]atch live changes made to a D2 source file in the default web browser:

`d2 --watch {{path/to/input_file.d2}} {{output_file.svg|output_file.png}}`

- Format a D2 source file:

`d2 fmt {{path/to/input_file.d2}}`

- List available themes:

`d2 themes`

- Use a different [t]heme for the output file (list available themes first to get the desired `theme_id`):

`d2 --theme {{theme_id}} {{path/to/input_file.d2}} {{output_file.svg|output_file.png}}`

- Make rendered diagrams look like hand [s]ketches:

`d2 --sketch true {{path/to/input_file.d2}} {{output_file.svg|output_file.png}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. Maybe, we could make the input into a path placeholder and also make the output filenames a bit more verbose.

File: pages/common/d2.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# d2
+
+> A modern diagram scripting language that turns text to diagrams.
+> More information: <https://d2lang.com/tour/intro/>.
```
Comment (kbdharun) [inline pages/common/d2.md:4]:
```suggestion
> More information: <https://d2lang.com/tour/man>.
```
Maybe, we can update this link to the man page of the command instead.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

**Review by Magrid0 — APPROVED** [review]
LGTM after the above suggestion

File: pages/common/d2.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# d2
+
+> A modern diagram scripting language that turns text to diagrams.
+> More information: <https://d2lang.com/tour/man>.
+
+- Compile and render a D2 source file into a SVG or PNG file:
```
Comment (acuteenvy) [inline pages/common/d2.md:6]:
```suggestion
- Compile and render a D2 source file into an SVG or PNG file:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**